pkg/solana/transmissions_cache.go

```diff
@@ -32,17 +26,9 @@ type ContractTracker struct {
 	// private key for the transmission signing
 	Transmitter TransmissionSigner
```

Comment:
`Transmitter` and `TxManager` can be removed from the `TransmissionsCache` and `StateCache` - leftovers from the `ContractTracker` split